msteams-platform/concepts/design/plan-responsive-tabs-for-teams-mobile.md

```diff
@@ -169,7 +169,7 @@ In the third stage, provide complete experience to your users. You can enable us
 * Delete, undo, and redo operations.
 * Access camera and microphone using JS SDK APIs. For more information on device capabilities, see [device capabilities overview](../device-capabilities/device-capabilities-overview.md).
 
-:::image type="content" source="../../assets/images/app-fundamentals/coauthoring-and-whiteboarding-apps-enablement.png" alt-text="Enable user experience by adding text shapes and quick notes and other capabilities.":::
+:::image type="content" source="../../assets/images/app-fundamentals/coauthoring-and-whiteboarding-apps-enablement.png" alt-text="The screenshot shows the steps to enable user experience by adding text shapes, quick notes, and other capabilities.":::
```

Comment:
So the article is about designing an app. These are some suggestions and best practices that some app developers will follow and some won't. The alt-text for each screenshot in this article should convey that the screenshot displays the specific good design that's shown in it.

Reply:
okay